hyperv.go

```diff
@@ -21,13 +21,15 @@ var HypervVendorIDSignatureString string
 
 var HypervInterfaceSignatureString string
 
+type HypervFeature uint64
```

Comment:
I'm not completely sure I see the benefits of these types, but they won't hurt either